aces/Loris
# [battery_manager] Resolved Permission Issues

## Brief summary of changes
This PR fixes the `getCenterId` function in `test.class.inc` so that it always passes a value to the UserSiteMatch filter by converting a centerId value of `null` from the database to an array of all sites.

#### Testing instructions (if applicable)
1. Create a user and do not assign it battery_manager view or edit permission.
2. Ensure that battery_manager cannot be accessed.
3. Assign user battery_manager view permission.
4. Ensure that battery_manager entries at user's site can be view, but there is no ability to add, activate, deactivate or edit the entries.
5. Give user battery_manager edit permission.
6. Ensure that entries can be added, activated, deactivated and edited.

#### Link(s) to related issue(s)
* Resolves #6682
